src/sequelize.ts

```diff
@@ -12,6 +12,6 @@ export const sequelize = new Sequelize({
   username: config.username,
   password: config.password,
   dialect: config.dialect,
-  logging: process.env.NODE_ENV?.toUpperCase() !== 'TEST',
+  logging: process.env.NODE_ENV?.toUpperCase() !== 'TEST' && console.log,
```

Comment:
💁🏻 this fixes a deprecation warning from sequelize where the value for `logging` should be either `false` or a function for logging

Reply:
Ah! I was wondering how to fix this 😄